float32 x
float32 y
float32 z
float32 doppler_velovity
float32 peak_value
bool brake_status
